crates/pop-cli/src/commands/install/mod.rs

```diff
@@ -141,13 +250,13 @@ async fn install_redhat(skip_confirm: bool) -> anyhow::Result<()> {
 		vec![
 			"install",
 			"-y",
-			"cmake",
-			"openssl-devel",
-			"git",
-			"protobuf",
-			"protobuf-compiler",
-			"clang",
-			"clang-devel",
+			&DEPENDENCIES::Cmake.to_string(),
+			&DEPENDENCIES::OpenSslDevel.to_string(),
+			&DEPENDENCIES::Git.to_string(),
+			&DEPENDENCIES::Protobuf.to_string(),
+			&DEPENDENCIES::ProtobufCompiler.to_string(),
+			&DEPENDENCIES::Clang.to_string(),
+			&DEPENDENCIES::ClangDevel.to_string(),
```

Comment:
Here and the above no rustup required?

Reply:
Yes. Rustup is installed in all environments. Is in the main function: https://github.com/r0gue-io/pop-cli/blob/alexd10s/pop-install/crates/pop-cli/src/commands/install/mod.rs#L92